### PR TITLE
re-run bot on review

### PR DESCRIPTION
**Pull request by alita-moore**

### CODE REVIEW

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - Filename .github/workflows/auto-merge-bot.yml is not in EIP format 'EIPS/eip-####.md'

**Review by alita-moore — COMMENTED** [review]
test

File: .github/workflows/auto-merge-bot.yml [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
 name: Auto-Merge Bot
+
+on:
+  pull_request_review:
+  pull_request_target:
+  deployment_status:
+  workflow_run:
+    workflows:
+      - travis-ci
```
Comment (MicahZoltu) [inline .github/workflows/auto-merge-bot.yml:10]: Does this actually work?

Comment (alita-moore) [reply .github/workflows/auto-merge-bot.yml:10]: no

**Review by MicahZoltu — APPROVED** [review]

**Pull request merged**